src/data/finset/image.lean

```diff
@@ -405,16 +407,18 @@ end
 
 lemma mem_range_iff_mem_finset_range_of_mod_eq [decidable_eq α] {f : ℤ → α} {a : α} {n : ℕ}
   (hn : 0 < n) (h : ∀ i, f (i % n) = f i) :
-  a ∈ set.range f ↔ a ∈ (finset.range n).image (λi, f i) :=
-suffices (∃ i, f (i % n) = a) ↔ ∃ i, i < n ∧ f ↑i = a, by simpa [h],
```

Comment:
Does the old proof no longer work?

Reply:
Old proof needs some lemmas in `data.int.order.basic`.

Reply:
Which ones?

Reply:
`int.mod_nonneg` and `int.mod_lt_of_pos`.

Reply:
I'm a little wary of this this split; those seem like pretty fundamental properties of `mod`, so I would prefer not to have to effectively reprove them to prove this lemma.

Can you split this into two PRs; one that moves the imports, and one that changes `int.order.basic` to `int.basic`?